test/homework1_test.exs

```diff
@@ -2,7 +2,27 @@ defmodule Homework1Test do
   use ExUnit.Case
   doctest Homework1
 
-  test "greets the world" do
-    assert Homework1.hello() == :world
+  test "addition" do
+    assert Homework1.calculate("1+1") === 2.0
+    assert Homework1.calculate("1+0") === 1.0
+    assert Homework1.calculate("-1+0") === -1.0
+  end
+
+  test "removal" do
+    assert Homework1.calculate("5-1") === 4.0
+    assert Homework1.calculate("1-5") === -4.0
+    assert Homework1.calculate("1-0") === 1.0
+  end
+
+  test "multiply" do
+    assert Homework1.calculate("2*5") === 10.0
+    assert Homework1.calculate("-2*5") === -10.0
+    assert Homework1.calculate("-2*0") === 0.0
+  end
+
+  test "divide" do
+    assert Homework1.calculate("2/5") === 0.4
+    assert Homework1.calculate("-2.5/5") === -0.5
+    assert Homework1.calculate("-2/1") === -2.0
```

Comment:
One should test the obvious corner case here, instead of coquettishly pretending that it does not exist. You’ve chosen to raise, ok, then test it, we have [`assert_raise/2`](https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#assert_raise/2) for that.